server/datastore/mysql/migrations/tables/20220809091020_AddWindowsUpdatesTable.go

```diff
@@ -0,0 +1,31 @@
+package tables
+
+import (
+	"database/sql"
+
+	"github.com/pkg/errors"
+)
+
+func init() {
+	MigrationClient.AddMigration(Up_20220809091020, Down_20220809091020)
+}
+
+func Up_20220809091020(tx *sql.Tx) error {
+	_, err := tx.Exec(`
+CREATE TABLE windows_updates (
+	id INT UNSIGNED NOT NULL AUTO_INCREMENT PRIMARY KEY,
+	host_id INT UNSIGNED NOT NULL,
+	date_epoch INT UNSIGNED NOT NULL,
```

Comment:
Any reason to not use `TIMESTAMP`? (Just curious.)

Reply:
Just to keep the implementation simple - all I care about is being able to 'order' updates based on installation date, an epoch works well enough for that.